lib/DDGC/Web/App/Base.pm

```diff
@@ -0,0 +1,33 @@
+package DDGC::Web::App::Base;
+
+use Import::Into;
+
+use strict;
+use warnings;
+use utf8;
+
+use Dancer2;
+use Dancer2::Plugin::DBIC;
+use DDGC::Web::Plugin::Config;
+use DDGC::Web::Plugin::Session;
+
+{   no warnings 'redefine';
+    sub import {
+        my ($caller, $filename) = caller;
+        for (
+          qw/
+            strict
+            warnings
+            utf8
+            Dancer2
+            Dancer2::Plugin::DBIC
+            DDGC::Web::Plugin::Config
+            DDGC::Web::Plugin::Session
+          /
+        ) {
+            $_->import::into($caller);
+        }
+    }
+};
+
+1;
```

Comment:
Just to note : This pattern does result in a deprecation warning (from Session plugin calling the keyword registered by the DBIC plugin), but that is on its way to being resolved elsewhere.

https://github.com/ironcamel/Dancer2-Plugin-DBIC/pull/9